changelog/11102.improvement.md

```diff
@@ -0,0 +1 @@
+Add `--endpoints` command line parameter to `rasa train` parser.
```

Comment:
We probably need to add this new info to the CLI docs too?

Reply:
Luckily the docs list the possible `train` parameters by calling `rasa train --help` so it will list the new param automatically.